images/build/installDotNetCore.sh

```diff
@@ -74,6 +74,6 @@ if [ "$INSTALL_TOOLS" == "true" ]; then
     chmod +x "$toolsDir/dotnet-dump"
     dotnet tool install --tool-path "$toolsDir" dotnet-counters
     chmod +x "$toolsDir/dotnet-counters"
-    dotnet tool install --tool-path "$toolsDir" dotnet-monitor --version 6.*
+    dotnet tool install --tool-path "$toolsDir" dotnet-monitor --version 7.*
```

Comment:
to my understanding, this is the default dotnet-version we use for dockerfiles. Then an individual dotnet version takes the dotnet-monitor version from its dockerfile.
So far, when we update the dotnet-version to a new one we set the new version as default. 
Please let me know if you think otherwise. Thanks

Reply:
@cormacpayne can you please confirm it?

Reply:
This `installDotNetCore.sh` script is used by a few build image Dockerfiles to pre-bake a version of .NET Core into the image, and some will set the `INSTALL_PACKAGES` environment variable that causes the above section to execute. Looking at previous PRs where we modify the version of `dotnet-monitor` installed, it appears that we bump the version to the same value across all runtime images using it and this script, so it doesn't appear this would cause any issues from a version-"mismtatch" point of view, but would still be worth testing from our side and letting the Antares team know of the change so they can test as well.

The one other thing I'd check is that all other runtime images that have a step to install `dotnet-monitor` are using the same version since it appears we're just updating the 6.0 and 7.0 images, but the 3.1 and 5.0 images also have a step to install` dotnet-monitor` that will be stuck on some version of `6.*` or `6.1.*` moving forward.

Reply:
Updated my comment above to differentiate `INSTALL_PACKAGES` and `INSTALL_TOOLS` -- it doesn't appear that `INSTALL_TOOLS` is being set anywhere in our repository, and it's not something that users can set as this is only relevant when the `installDotNetCore.sh` script is ran when constructing the images, so the above section is never currently executed.

Historically, when this was [first added](https://github.com/microsoft/Oryx/commit/a8047325620bddbf82b210a52d8caeecbd939239#diff-d2d91164bba96021bc9e98f865d8f0073c633c60aca54d7eafafcedd8ac29102), we were setting `INSTALL_TOOLS` in the `latest` Dockerfile when installing .NET Core 1.1 and 3.0. I think around [this commit](https://github.com/microsoft/Oryx/commit/58e488370675955a21da4c775dfa3efc356873d2#) was when the `INSTALL_TOOLS` environment variable stopped being set in the Dockerfile, but I can't find any reason for why.

If we haven't used it in ~3 years, it may not be a bad idea to remove this snippet from the `installDotNetCore.sh` script just so we're not worried about bumping versions here moving forward.

Reply:
@cormacpayne  Thanks for the feedback. 
I will remove the whole code-block started from the if condition- `if [ "$INSTALL_TOOLS" == "true" ]; then`
https://github.com/microsoft/Oryx/blob/main/images/build/installDotNetCore.sh#:~:text=if%20%5B%20%22%24INSTALL_TOOLS,fi

Reply:
@pauld-msft yah dotnet-monitor is tool is used only in the runtime images.